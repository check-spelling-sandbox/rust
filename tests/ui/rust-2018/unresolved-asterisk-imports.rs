// edition:2018

use nonexistent_crate::*; //~ ERROR unresolved import `nonexistent_crate
use std as foo;

fn main() {}
