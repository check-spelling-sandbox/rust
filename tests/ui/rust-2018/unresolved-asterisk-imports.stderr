error[E0432]: unresolved import `nonexistent_crate`
  --> $DIR/unresolved-asterisk-imports.rs:3:5
   |
LL | use nonexistent_crate::*;
   |     ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `nonexistent_crate`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
